### PR TITLE
Readme file updates

### PR DESCRIPTION
**Pull request by arun-msft**
Updated deployment steps in readme file.

### CODE REVIEW

File: TeamsSampleHub/SampleHubHTMLCSSVersion/README.md [hunk 1]
```diff
@@ -176,7 +176,10 @@ Deployment
 3. On the overview page, copy and save the **Application (client) ID, Directory (tenant) ID**. You’ll need those later when updating your Teams application manifest and in the appsettings.json.
 4. Under **Manage**, select **Expose an API**. 
 5. Select the **Set** link to generate the Application ID URI in the form of `api://{AppID}`. Insert your fully qualified domain name (with a forward slash "/" appended to the end) between the double forward slashes and the GUID. The entire ID should have the form of: `api://fully-qualified-domain-name.com/{AppID}`
-    * ex: `api://subdomain.example.com/00000000-0000-0000-0000-000000000000`.
+    * ex: `api://%appDomain%.azurewebsites.net/00000000-0000-0000-0000-000000000000`.
+
+        * `Note, Application ID URI can be set post deployment also, if the appDomain is not available yet.
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHubHTMLCSSVersion/README.md]: `Note, Application ID URI can be set post deployment also, if the appDomain is not available.

File: TeamsSampleHub/SampleHubHTMLCSSVersion/README.md [hunk 2]
```diff
@@ -187,7 +190,10 @@ Deployment
 9. Ensure that **State** is set to **Enabled**
 10. Select **Add scope**
     * The domain part of the **Scope name** displayed just below the text field should automatically match the **Application ID** URI set in the previous step, with `/access_as_user` appended to the end:
-        * `api://subdomain.example.com/00000000-0000-0000-0000-000000000000/access_as_user`
+        * `api://%appDomain%.azurewebsites.net/00000000-0000-0000-0000-000000000000/access_as_user.
+
+            * `Note, Redirect URI can be set post deployment also, if the appDomain is not available yet
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHubHTMLCSSVersion/README.md]: Remove "Yet"

File: TeamsSampleHub/SampleHubReactVersion/README.md [hunk 3]
```diff
@@ -176,7 +176,10 @@ Deployment
 3. On the overview page, copy and save the **Application (client) ID, Directory (tenant) ID**. You’ll need those later when updating your Teams application manifest and in the appsettings.json.
 4. Under **Manage**, select **Expose an API**. 
 5. Select the **Set** link to generate the Application ID URI in the form of `api://{AppID}`. Insert your fully qualified domain name (with a forward slash "/" appended to the end) between the double forward slashes and the GUID. The entire ID should have the form of: `api://fully-qualified-domain-name.com/{AppID}`
-    * ex: `api://subdomain.example.com/00000000-0000-0000-0000-000000000000`.
+    * ex: `api://%appDomain%.azurewebsites.net/00000000-0000-0000-0000-000000000000`.
+
+        * `Note, Application ID URI can be set post deployment also, if the appDomain is not available yet.`
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHubReactVersion/README.md]: Remove "yet"

File: TeamsSampleHub/SampleHubReactVersion/README.md [hunk 4]
```diff
@@ -187,7 +190,10 @@ Deployment
 9. Ensure that **State** is set to **Enabled**
 10. Select **Add scope**
     * The domain part of the **Scope name** displayed just below the text field should automatically match the **Application ID** URI set in the previous step, with `/access_as_user` appended to the end:
-        * `api://subdomain.example.com/00000000-0000-0000-0000-000000000000/access_as_user`
+        * `api://%appDomain%.azurewebsites.net/00000000-0000-0000-0000-000000000000/access_as_user`
+
+            * `Note, Application ID URI can be set post deployment also, if the appDomain is not available yet.`
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHubReactVersion/README.md]: Remove "yet"

File: TeamsSampleHub/SampleHubReactVersion/README.md [hunk 5]
```diff
@@ -228,6 +234,14 @@ Deployment
 -   Download the Sample Hub code from the repository and open in Visual studio
     explorer. 
 
+-   Complete the necessary app setting values:
+
+        1. Azure client Id (App id).
+        2. Deep Link URL and all required app Ids which we need to deep link.
+        3. If you have SharePoint Framework app then provide the SharePoint client Id and List Id.  
+        4. Update the app ids in the appsettings.config file in the api before deployment
+	    5. Update the apiendpoint in the links.tsx file in ClientApp folder
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHubReactVersion/README.md:243]: Fix notation

**Review by aamini-ms — APPROVED** [review]

**Comment by arun-msft** [comment]
Updated deployment steps in readme file.

**Pull request closed without merging**